packages/tsx-editor/src/demo/App.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { PrimaryButton, Stack, Label, mergeStyleSets } from 'office-ui-fabric-react';
+// import { ITranspileOutput } from '../transpiler/transpile.types';
+
+const classNames = mergeStyleSets({
+  code: {
+    fontFamily: 'monospace',
+    fontSize: 13,
+    lineHeight: '1.5'
+  },
+  renderSection: {
+    backgroundColor: 'red'
+  },
+  error: {
+    backgroundColor: '#FEF0F0',
+    color: '#FF5E79'
+  },
+  editor: {
+    width: 800,
+    height: 500
+  }
+});
+
+interface IAppState {
+  code: string;
+  JScode: string;
+  error?: string;
+  fontSize?: string;
+  editorHidden?: boolean;
+  editor?: JSX.Element;
+  currentTime: number;
+}
+
+export class App extends React.Component {
+  public state: IAppState = {
+    code: '',
+    JScode: '',
+    editor: undefined,
+    currentTime: 0,
+    editorHidden: true
+  };
+
+  public render() {
+    const editor = (
+      <Stack style={{ backgroundColor: 'lightgray' }} gap={4}>
+        {!this.state.editorHidden && this.state.editor}
```

Comment:
Since you check `this.state.editorHidden` later:
```suggestion
        {this.state.editor}
```